const array = [];

try {
  console.log(array[10].umaPropriedade);
} catch (error) {
  console.log("Dei erro :( ! MAs eu tratei :)");
}
console.log("Eu tam√©m passei");
